course_content/iris_course/6.Data_Processing.ipynb

```diff
@@ -69,11 +104,30 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "a1b = iris.load_cube(iris.sample_data_path('A1B_north_america.nc'))\n",
-    "e1 = iris.load_cube(iris.sample_data_path('E1_north_america.nc'))\n",
+    "#\n",
+    "# edit space for user code\n",
+    "#"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# SAMPLE SOLUTION\n",
+    "# %load solutions/iris_exercise_6.1b"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----\n",
     "\n",
-    "print(e1.summary(True))\n",
-    "print(a1b)"
+    "Although a cube's units can be freely set to any valid unit, the calculation of result units and compatibility checking is built into the arithmetic operations.\n",
+    "\n",
+    "For example:"
```

Comment:
It might be worth mentioning that you are creating new cubes.

so, this could be something like:
```
For example, let's create two new cubes, one with units of feet and one with the units of days.
```